backend/app/sql/total_pop_udf.sql

```diff
@@ -5,8 +5,10 @@ DECLARE
     sql_query TEXT;
     total_pop_column_name TEXT;
 BEGIN
-    SELECT gerrydb_table INTO gerrydb_table_name
+    SELECT districtrmap.parent_layer INTO gerrydb_table_name
```

Comment:
@nofurtherinformation this is the issue. Should be `districtrmap.gerrydb_table_name INTO gerrydb_table_name` not the parent layer